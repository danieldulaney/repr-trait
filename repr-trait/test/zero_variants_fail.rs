#[derive(repr_trait::PrimitiveRepr)]
enum ZeroVariants {
}

fn main() {

}
