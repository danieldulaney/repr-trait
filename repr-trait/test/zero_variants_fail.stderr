error: proc-macro derive panicked
 --> test/zero_variants_fail.rs:1:10
  |
1 | #[derive(repr_trait::PrimitiveRepr)]
  |          ^^^^^^^^^^^^^^^^^^^^^^^^^
  |
  = help: message: Can't derive PrimitiveRepr on a zero variant enum
